test_autoflake.py

```diff
@@ -1224,6 +1224,43 @@ def test_exclude(self):
         finally:
             shutil.rmtree(temp_directory)
 
+    def test_with_ignore_init_module_imports_flag(self):
+        # Need a temp directory in order to specify file name as __init__.py
+        temp_directory = tempfile.mkdtemp(dir='.')
+        try:
+            with open(
+                 os.path.join(temp_directory, '__init__.py'), 'w') as output:
+                output.write('import re\n')
+
+            p = subprocess.Popen(
+                list(AUTOFLAKE_COMMAND) +
+                [temp_directory + '/__init__.py',
```

Comment:
here is the literal `/` , which probably should be using `os.path.join` - yell out if I have forgotten something which means it needs to be a `/`

Reply:
Yep, fixed by using the variable mentioned earlier!